src/main/java/org/jenkinsci/plugins/periodicbackup/PeriodicBackupLink.java

```diff
@@ -255,7 +255,7 @@ public DescriptorImpl getDescriptor() {
     public static final class DescriptorImpl extends Descriptor<PeriodicBackupLink> {
 
         public String getDisplayName() {
-            return null; // unused
+            return ""; // unused

```

Comment:
Without this `spotbugs` would fail  : 
```[INFO] --- spotbugs-maven-plugin:4.0.4:check (spotbugs) @ periodicbackup ---
[INFO] BugInstance size is 1
[INFO] Error size is 0
[INFO] Total bugs: 1
[ERROR] High: org.jenkinsci.plugins.periodicbackup.PeriodicBackupLink$DescriptorImpl.getDisplayName() may return null, but is declared @Nonnull [org.jenkinsci.plugins.periodicbackup.PeriodicBackupLink$DescriptorImpl] At PeriodicBackupLink.java:[line 258] NP_NONNULL_RETURN_VIOLATION
[INFO] 
```